docs/manual/projects/scm/git.md

```diff
@@ -50,6 +59,10 @@ Changing this value modifies the file definition and files need to be pushed aga
 **SSH: Strict Host Key Checking**: If yes, require remote host SSH key is defined in the `~/.ssh/known_hosts` file, otherwise do not verify.
 
 **SSH Key Storage Path** (Optional): A Storage Key path containing the private key to be used with git authentication.
+:::tip
+Use the following command to generate the ssh key pair:
+`ssh-keygen -t rsa -m PEM -f rundeck-qa-key -C ""`
```

Comment:
Does this need "qa" in the key name?

Reply:
I thought I changed it, sorry. Let me update